src/javascripts/application.js

```diff
@@ -15,7 +15,7 @@ Example.init('.js-example__frame')
 Tabs.init('.js-example')
 
 // Add copy to clipboard to code blocks inside tab containers
-Copy.init('.app-tabs__container pre')
+new Copy('.app-tabs__container pre').init()
```

Comment:
I think to mirror GOV.UK Frontend the Copy constructor only accepts once instance of the place to enhance.

So we'd move the nodeListForEach outside into application.js.

It's also worth looking into if we could use `data-module="app-copy"`?

Reply:
for the data- attribute i'd need to get #504 merged in so i can attach the attribute to it. i don't want to attach the attribute to each example container